src/app/plugins/connectfour.plugin.ts

```diff
@@ -144,6 +144,11 @@ class ConnectFourGame {
   private _playingLion: boolean;
   private _aiDepth = 4;
 
+  // messy initialization since typescript won't let me use -1 or null
```

Comment:
`ReturnType<...> | null` doesnt work here?

Reply:
Good idea. Used `undefined` instead of `null`, since that matches `clearTimeout()`'s signature.